src/CSLOLToolsImpl.h

```diff
@@ -75,6 +75,9 @@ public slots:
 
     CSLOLToolsImpl::CSLOLState getState();
     QString getLeaguePath();
+#if __APPLE__
+    int isUnderRosetta();
+#endif
```

Comment:
See my other comments, this should got to other header file.
Additional C++ has builtin ``bool`` type instead of int.

Reply:
:+1: